packages/api/package.json

```diff
@@ -23,6 +23,9 @@
   },
   "typings": "./lib/index.d.ts",
   "license": "MIT",
+  "bin": {
```

Comment:
Could also just be:

`"bin": "./bin/run-server.js"``, then it can be called via `npx @wepublish/api`